openvr/LICENSE

```diff
@@ -1,27 +0,0 @@
-Copyright (c) 2015, Valve Corporation
```

Comment:
The license was not removed. I removed the existing `openvr` subdirectory entirely and replaced it with a submodule pointing to the official repository. This will help keep the size of this repository down and make future updates easier.